tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal

```diff
@@ -161,10 +161,10 @@ type Employee record {|
     string dept;
 |};
 
-type Candidate record {
+type Candidate record {|
     string name;
     never university?;
-};
+|};
```

Comment:
May make sense to leave this as is and update the target type, since `never university?;` is no longer needed here since the record is closed.